src/scss/includes/direction.scss

```diff
@@ -1,6 +1,32 @@
-.direction_panel {
-  background: url("../images/panel_top.svg") #f4f6fa repeat-x !important;
-  background-size: 100% 5px;
+.direction-panel {
+  background-color: $background;
+  @include card_shadow();
+  width: 400px;
+
+  position: absolute;
+
+  @media (min-width: 641px) {
+    @include panel_radius();
+  }
+
+  @media (max-width: 640px) {
+    position: fixed;
+    min-height: 42px;
+    width: 100%;
+    top: 0;
+    left: 0;
+    z-index: 3;
+
+    .itinerary_title {
```

Comment:
This can be removed as the same exact rules are defined below outside of the media query :)

Reply:
This selector can still be removed.